at-hoangtran/Week4/Final/Controllers/session_controller.rb

```diff
@@ -0,0 +1,64 @@
+require_relative '../Migrate/connect_db'
+require_relative '../Models/employee_model'
+require 'pry'
+require 'bcrypt'
+class SessionController
+  $session = {}
+  def login
+    system('clear')
+    p '=============LOGIN============='
+    print 'User Name: '
+    user_name = gets.chomp
+    print 'Password: '
+    password = gets.chomp
+    login = EmployeeModel.login(user_name, password)
+    if login
+      $session[:user] = user_name
+      p 'Login Success'
+    else
+      p 'Invalid Username or Password'
+    end
+
+    binding.pry
+  end
+
+  def self.register
+    system('clear')
+    p '===========REGISTER============'
+    print 'Name: '
+    name = gets.chomp
+    loop do
+      print 'User Name: '
+      @user_name = gets.chomp
+      check = EmployeeModel.check_user(@user_name)
+      if check.count == 0
+        break
+      else
+        p 'User Exist !'
+      end
+    end
+    print 'Password: '
+    password = BCrypt::Password.create(gets.chomp)
+    print 'Team Id: '
+    team_id = gets.chomp.to_i
+    register = EmployeeModel.register(name, team_id, @user_name, password)
+    'Register Success'
+    EmployeeModel.login(user_name, password)
+  end
+
+  def self.forgot_password
+    system('clear')
+    p '=======FORGOT PASSWORD========='
+    print 'User Name: '
+    user_name = gets.chomp
+    check = EmployeeModel.check_user(user_name)
+    if check.count == 0
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/8bc115c94428b11d6e3406f6ef06575baf3e6ffc/at-hoangtran/Week4/Final/Controllers/session_controller.rb#L55"></span>Style/NumericPredicate: Use <code>check.count.zero?</code> instead of <code>check.count == 0</code>. (https://github.com/rubocop-hq/ruby-style-guide#predicate-methods)</td>
    </tr>
  </tbody>
</table>